packages/edit-site/src/store/selectors.js

```diff
@@ -135,11 +135,7 @@ export function getHomeTemplateId( state ) {
 }
 
 function getCurrentEditedPost( state ) {
-	return state.editedPost[ state.editedPost.length - 1 ] || {};
-}
-
-function getPreviousEditedPost( state ) {
-	return state.editedPost[ state.editedPost.length - 2 ] || {};
+	return state.editedPost;
```

Comment:
@kevin940726: in the interest of YAGNI, can we remove `getCurrentEditedPost` and let the selectors access state directly? As per before https://github.com/WordPress/gutenberg/pull/34732/files#diff-94ccd649d2e3a0efc85ceb76fc0ebf13e6b721cedec21edc917064fe4cb2c79bL143.

Reply:
Yeah sure. We needed it because we had `getPreviousEditedPost` which made it confusing. Now that we don't have that we can remove this intermediate function.